lib/iiif/v3/presentation/annotation.rb

```diff
@@ -3,22 +3,35 @@ module V3
     module Presentation
       class Annotation < IIIF::V3::AbstractResource
 
-        TYPE = 'Annotation'
+        TYPE = 'Annotation'.freeze
 
         def required_keys
-          super + %w{ motivation }
+          super + %w{ id motivation target }
```

Comment:
Annotation itself isn't in the 3.0 API;  it has its own spec:  http://www.openannotation.org/spec/core/   In the spec it is `hasTarget` but our JSON expresses it as `target`

I learned about Annotations for the Triannon linked data project I did a while ago;  but mostly I looked at examples -- our purl, Tom Crane's, etc.